src/core/lv_obj_pos.c

```diff
@@ -282,7 +282,7 @@ void lv_obj_mark_layout_as_dirty(lv_obj_t * obj)
     scr->scr_layout_inv = 1;
 
     /*Make the display refreshing*/
-    lv_display_t * disp = lv_obj_get_disp(scr);
+    lv_display_t * disp = lv_obj_get_display(scr);
```

Comment:
Need add `lv_obj_get_display ` to api map.

Reply:
Added